### DISCUSSION TITLE
Support for application which needs physical system and can not support VM

### DISCUSSION THREAD

**Discussion by dellclientopensource** [microsoft/winget-pkgs#115499] [Q&A]
I am trying to upload Dell command monitor in winget but requires physical system to install and run. As VM does not have a WMI-ACPI compliant BIOS, it fails at validation from winget. And we are blocked to update this in winget store.
Is it possible to enable hardware support for winget validation and get approval for such application which needs hardware for installation and not VM?

**Comment by Trenly** [comment]
This request is already being tracked -
* https://github.com/microsoft/winget-cli/issues/1417
